### PR TITLE
measure-cloud: support the release metanames

### PR DESCRIPTION
**Pull request by paride**
Support the Ubuntu release metanames: lts, stable, devel, latest.
They are resolved to actual release names using ubuntu-distro-info(1).

### CODE REVIEW

File: boot-speed/clouds/measure-cloud.py [hunk 1]
```diff
@@ -323,6 +331,10 @@ def gen_archivename(metadata):
     arcname = cloud + "-" + inst_type + "-" + release + "_" + date
     return arcname
 
+def metaname2release(metaname):
```
Comment (powersj) [inline boot-speed/clouds/measure-cloud.py]: Have you looked at python3-distro-info?

```
import distro_info
    try:
        devel = distro_info.UbuntuDistroInfo().devel()
    except distro_info.DistroDataOutdated:
        devel = distro_info.UbuntuDistroInfo().stable()

    lts = distro_info.UbuntuDistroInfo().lts()
```

Comment (paride) [reply boot-speed/clouds/measure-cloud.py]: Now I want again to implement it in pycludlib. It shouldn't be difficult really.

Comment (paride) [reply boot-speed/clouds/measure-cloud.py]: Mmh no, I don't want to, as python3-distro-info is too Debian/Ubuntu specific, and I don't think we want the library to depend on it. It's not even in PyPI (`distroinfo` is a different thing).

Comment (powersj) [reply boot-speed/clouds/measure-cloud.py]: yeah for pycloudlib I would be -1 on it, but for our test scripts it can definitely include it.

Comment (paride) [reply boot-speed/clouds/measure-cloud.py]: Done (a bit differently).

**Pull request merged**